coalib/bearlib/abstractions/Linter.py

```diff
@@ -444,9 +434,17 @@ def process_output_regex(
                                 "result_message")
                     if key in options}
 
-                process_output = partialmethod(
+                _processing_function = partialmethod(
                     process_output_regex, **_process_output_args)
 
+            def process_output(self, output, filename, file):
```

Comment:
please make a good documentation comment, people will look here when overriding this function